### PR TITLE
Feature: Add rake task to fetch currency current price

### PR DESCRIPTION
**Pull request by chumaknadya**

### CODE REVIEW

File: lib/tasks/fetch.rake [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+namespace :fetch do
+  desc 'Fetch currency price'
+  task :price, %i[quote] => [:environment] do |_, args|
+    Currency.find_in_batches(batch_size: 50) do |group|
+      group.each do |record|
+        # API call to external resource domain to get current currency price
+        response = Faraday.get(ENV['EXTERNAL_RESOURCE_DOMAIN'], { code: record.code, quote: args.quote })
```
Comment (mnaichuk) [inline lib/tasks/fetch.rake]: let's do with rake task params and u can use default from env
and if not specified both you log error and quit job

File: lib/tasks/fetch.rake [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+namespace :fetch do
+  desc 'Fetch currency price'
+  task :price, %i[quote external_resource_domain] => [:environment] do |_, args|
+    Currency.find_in_batches(batch_size: 50) do |group|
+      group.each do |record|
+        url = args.external_resource_domain || ENV['EXTERNAL_RESOURCE_DOMAIN']
+        raise 'There is no external resource domain configured' unless url.present?
+
+        # API call to external resource domain to get current currency price
+        response = Faraday.get(url, { code: record.code, quote: args.quote })
```
Comment (mnaichuk) [inline lib/tasks/fetch.rake:14]: you should use `url` here

File: lib/tasks/fetch.rake [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :fetch do
+  desc 'Fetch currency price'
+  task :price, %i[quote external_resource_domain] => [:environment] do |_, args|
+    Currency.find_in_batches(batch_size: 50) do |group|
+      group.each do |record|
+        url = args.external_resource_domain || ENV['EXTERNAL_RESOURCE_DOMAIN']
```
Comment (calj) [inline lib/tasks/fetch.rake]: let's name it EXTERNAL_CURRENCIES_SERVICE

File: lib/tasks/fetch.rake [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :fetch do
+  desc 'Fetch currency price'
+  task :price, %i[quote external_currencies_service] => [:environment] do |_, args|
+    Currency.find_in_batches(batch_size: 50) do |group|
```
Comment (mnaichuk) [inline lib/tasks/fetch.rake:8]: Add please TODO to add ability to take price from platform market

**Review by calj — APPROVED** [review]

**Pull request merged**